packages/calling-stateful-client/src/DeviceManagerDeclarative.ts

```diff
@@ -70,12 +70,12 @@ class ProxyDeviceManager implements ProxyHandler<DeviceManager> {
   };
 
   private videoDevicesUpdated = async (): Promise<void> => {
-    this._context.setDeviceManagerCameras(await this._deviceManager.getCameras());
+    this._context.setDeviceManagerCameras(dedupeVideoDevices(await this._deviceManager.getCameras()));
```

Comment:
These dedupes are intended to be temporary because duplicates are being returned from the SDK currently

Reply:
Please add a comment in the code with a TODO(...) referencing the bug / task against SDK that will remove the duplicates.

Reply:
Actually, better just add the bug reference near the definition of `dedupeAudioDevices` on line 145

Reply:
Unfortunately, I still haven't got confirmation from Calling that this is a bug and they have not logged a ticket. But I did add TODO to the deduper function for future reference.